daffodil-runtime1/src/main/scala/edu/illinois/ncsa/daffodil/processors/parsers/Parser.scala

```diff
@@ -104,10 +68,17 @@ trait Parser
 
   protected def parse(pstate: PState): Unit
 
-  final def parse1(pstate: PState): Unit = {
```

Comment:
The below callback to the PState method parse1 isn't necessary. Merge with parse2 below.